### PR TITLE
Ria 3313 ftpa submitted notification

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3313

### Change description ###
CHanges to send HO notification on FTPA submission

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/CourtType.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public enum CourtType {
     FIRST_TIER,
-    FTPA,
-    UTPA;
-
+    UPPER_TRIBUNAL,
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/CourtType.java:5]: Do these enum values need to be tested ?

**Review by Lee365 — COMMENTED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/CourtType.java (hunk 1 above)
Comment (jkumar-moj) [reply src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/CourtType.java:5]: I don't think it is needed. They just represents possible values for any future implementations.

File: src/functionalTest/resources/scenarios/RIA-3313-notification-for-ftpa-appellant-ftpa-decided.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "description": "RIA-3313 FTPA appellant in time application submission notification - FTPA submitted state",
```
[the remark this answers is not in the document]
Comment (jkumar-moj) [reply src/functionalTest/resources/scenarios/RIA-3313-notification-for-ftpa-appellant-ftpa-decided.json]: description changed to reflect the right state

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Comment by jkumar-moj** [comment]
Reopening the PR as it was closed by stale bot

**Pull request merged**